src/components/Speciality/Books.js

```diff
@@ -10,6 +10,9 @@ import breakpoint from 'styled-components-breakpoint'
 import { SectionTitle, CardTitle, BodyPrimary } from '../Typography'
 import StyledLink from '../Common/StyledLink'
 
+// TODO - should come from contentful
+const moreBooksLink = 'http://nodepatternsbooks.com/index.html'
```

Comment:
I reckon any time we add a TODO to the codebase, we should document it in a trello ticket. What do you guys think?

In this situation, I guess it could go into a checklist in the content model review?

Reply:
Agreed 👍